conda_smithy/configure_feedstock.py

```diff
@@ -720,9 +723,7 @@ def _render_ci_provider(
                 permit_undefined_jinja=True,
                 finalize=False,
                 bypass_env_check=True,
-                channel_urls=forge_config.get("channels", {}).get(
-                    "sources", []
-                ),
+                channel_urls=channel_sources,
```

Comment:
I am not sure about this change. I would appreciate some more info if someone knows!